infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -145,17 +212,18 @@ def restore_database(backup_fpath):
 
 
 def main():
+    print("System args:", sys.argv)
     print("Running database-backup process with subcommand", sys.argv[1])
     if sys.argv[1] == "backup":
```

Comment:
Same thing. I'd extract the function call so we have a variable name as extra documentation of what's going on.

Reply:
Sorry, more context: I don't know that I would have been able to intuit that the return value of `make_object_name()` is setting the location in S3 to save the backup file to. Can you also label the parameter name that this is getting passed in for?